siroc.config.ts

```diff
@@ -0,0 +1,37 @@
+import { defineSirocConfig } from 'siroc'
+
+import { terser } from 'rollup-plugin-terser'
+import babel from '@rollup/plugin-babel'
+
+export default defineSirocConfig({
+  hooks: {
+    'build:extendRollup' (_pkg, { rollupConfig }) {
+      rollupConfig.push({
+        input: './src/lazy-script.js',
+        output: [
+          {
+            file: './dist/lazy-script.js',
+            format: 'iife',
+            plugins: [
+              terser()
+            ]
+          }
+        ],
+        plugins: [
+          babel({
+            babelHelpers: 'inline',
+            presets: [
+              [
+                '@babel/preset-env', {
+                  targets: {
+                    ie: '9'
```

Comment:
We might increase it since IntersectionObserver is never supported in IE. (safari 12 might be an option)